Contribution instruction to README, --skip-missing-interpreters tip, and ditch Makefile

ArnaudBrousseau opened the pull request:
Switches from tox to tox2, and adds --skip-missing-interpreters (not everybody has pypy/pypy3 installed on their laptop.)

ArnaudBrousseau commented:
@asottile shipit?

asottile commented:
tox2 is a yelpism.  I'd instead suggest applying this just internally in the jenkins configuration.  Ping me in irc and I can show you a few examples where we do this

ArnaudBrousseau commented:
Didn't know tox2 was a yelpism. Whoops. After discussion, removed the Makefile, it's not doing anything worth keeping (Travis just does "tox"). Added some contribution instructions to the README instead.

asottile commented on README.md:
````diff
@@ -53,6 +53,16 @@ together by a common "`wrapper`" node.
+(use tox2 if you're running on Yelp's internal servers)
+```
+`--skip-missing-interpreters` will allow you to skip missing interpreters.
+Those will be tested by TravisCI when you issue a pull request.
````
nitpick: Travis CI

asottile commented on README.md:
````diff
@@ -53,6 +53,16 @@ together by a common "`wrapper`" node.
+
+```
+tox --skip-missing-interpreters
+(use tox2 if you're running on Yelp's internal servers)
````
I wouldn't bother mentioning internal-specific instructions, shrugs

ArnaudBrousseau commented:
boatit?

asottile commented:
:horse::boat:s
